parts/kubernetesmastercustomscript.sh

```diff
@@ -95,6 +95,12 @@ function ensureKubectl() {
     fi
 }
 
+function downloadUrl () {
```

Comment:
Ok, @amanohar had asked for the same thing a few comments above. I'll add it, but again, there are no test cases for Calico. For regression testing, all I can do is to check if the addons are downloaded to that file path. I have no other means to test if something is broken because of this change.